.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
```

Comment:
pick a better name for the workflow

Reply:
It's a very common name and known as "continuous integration"